### PR TITLE
[AIDAPP-205]: Introduce a badge that displays the current status (e.g., published) for a knowledge base article.

### PR DESCRIPTION
**Pull request by ketan-canyon**
### Ticket(s) or GitHub Issue

- https://canyongbs.atlassian.net/browse/AIDAPP-205

### Technical Description

add badge for status.

### Any deployment steps required?

No

### Are any Feature Flags Added?

No

_______________________________________________

### CODE REVIEW

File: app-modules/knowledge-base/resources/views/filament/pages/badges.blade.php [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+      in the Elastic License 2.0.
+
+    For more information or inquiries please visit our website at
+    <https://www.canyongbs.com> or contact us via email at legal@canyongbs.com.
+
+</COPYRIGHT>
+--}}
+
+<div class="flex flex-col items-start">
+    <span>Status:</span>
```
Comment (Orrison) [inline app-modules/knowledge-base/resources/views/filament/pages/badges.blade.php:36]: I am not sure adding this text here was a part of the design scope in the ticket. Correct me if I am wrong, though.

If not, we should remove it. But if you feel strongly that it is a good idea to have it here, then please propose the change to Product within the ticket.

File: app-modules/knowledge-base/resources/views/filament/pages/badges.blade.php [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+    <x-filament::badge class="gap-2">
+        {{ $getRecord()->status ? $getRecord()->status->name : '' }}
```
Comment (Orrison) [inline app-modules/knowledge-base/resources/views/filament/pages/badges.blade.php:39]:
```suggestion
        {{ $getRecord()?->status?->name ?? '' }}
```

File: app-modules/knowledge-base/src/Filament/Resources/KnowledgeBaseItemResource/Pages/EditKnowledgeBaseItem.php [hunk 3]
```diff
@@ -87,6 +88,8 @@ public function form(Form $form): Form
                                     }),
                             ),
                     ]),
+                View::make('knowledge-base::filament.pages.badges')
+                    ->visible(fn (Model $record): bool => ($record->status) ? true : false),
```
Comment (Orrison) [inline app-modules/knowledge-base/src/Filament/Resources/KnowledgeBaseItemResource/Pages/EditKnowledgeBaseItem.php:92]:
```suggestion
                    ->visible(fn (Model $record): bool => ! empty($record->status)),
```

**Review by Orrison — CHANGES_REQUESTED** [review]

File: app-modules/knowledge-base/resources/views/filament/pages/badges.blade.php (hunk 1 above)
Comment (ketan-canyon) [reply app-modules/knowledge-base/resources/views/filament/pages/badges.blade.php]: Done

File: app-modules/knowledge-base/resources/views/filament/pages/badges.blade.php (hunk 2 above)
Comment (ketan-canyon) [reply app-modules/knowledge-base/resources/views/filament/pages/badges.blade.php]: Done

File: app-modules/knowledge-base/src/Filament/Resources/KnowledgeBaseItemResource/Pages/EditKnowledgeBaseItem.php (hunk 3 above)
Comment (ketan-canyon) [reply app-modules/knowledge-base/src/Filament/Resources/KnowledgeBaseItemResource/Pages/EditKnowledgeBaseItem.php]: Done

**Comment by Orrison** [comment]
Hey @ketan-canyon, this is just a reminder per our discussion earlier this week here. I noticed you force-pushed here. Was there a particular reason you needed to do so? i.e. you needed to rebase?

If it was just based there were changed in this remote due to CI. You most likely should have pulled those changes down first before making your local changes to avoid needing to rebase and/or force push.

**Review by Orrison — APPROVED** [review]

**Comment by ketan-canyon** [comment]
> Hey @ketan-canyon, this is just a reminder per our discussion earlier this week here. I noticed you force-pushed here. Was there a particular reason you needed to do so? i.e. you needed to rebase?
> 
> If it was just based there were changed in this remote due to CI. You most likely should have pulled those changes down first before making your local changes to avoid needing to rebase and/or force push.

Hi @Orrison 
I already taken pull.
Also I have also done rebase because there is changes in main branch so I think force push is there.

**Pull request merged**